### ISSUE TITLE
[RFE] Notifications via apprise

### ISSUE THREAD

**Issue by badnetmask** [mattwebbio/orbital-sync#264]
### What feature would you like added?

I would like to be able to send notifications using [apprise](https://github.com/caronc/apprise). This is a great app, very flexible, that allows sending notifications via multiple channels.

### Version

1.5.7

**Comment by mattwebbio** [comment]
I agree that it would be awesome to support further notification mediums. I can't promise it'll be AppRise, but I'll certainly look into it
